### PR TITLE
Zhoubo01 es

### PR DESCRIPTION
**Pull request by TomorrowIsAnOtherDay**
### This PR:
- use new APIs for parallel training.
- add performance on Mujoco benchmarks.
- add Documentation link in README.md of the main page.

### CODE REVIEW

File: examples/ES/README.md [hunk 1]
```diff
@@ -20,18 +20,14 @@ TODO
 
 ### Distributed Training
 
-#### Learner
-```sh
-python learner.py 
-```
+To replicate the performance reported above, we encourage training with 96 CPUs. 
+If you haven't created a cluster before, enter the following command to create a cluster with 96 CPUs. For more information about the cluster, please refer to our [documentation](https://parl.readthedocs.io/en/latest/parallel_training/setup.html).
```
Comment (zenghsh3) [inline examples/ES/README.md]: should we add `python learner.py`?

Comment (TomorrowIsAnOtherDay) [reply examples/ES/README.md]: Added. Note that I have renamed `learner.py` to `train.py`

File: examples/ES/README.md [hunk 2]
```diff
@@ -20,18 +20,21 @@ TODO
...
+If you haven't created a cluster before, enter the following command to create a cluster. For more information about the cluster, please refer to our [documentation](https://parl.readthedocs.io/en/latest/parallel_training/setup.html).
+
+```bash
+xparl start --port 8037 --cpu_num 96
 ```
 
-#### Actors
-```sh
-sh run_actors.sh
+Then we can start the distributed training by running train.py.
```
Comment (zenghsh3) [inline examples/ES/README.md]: by running:

**Review by zenghsh3 — APPROVED** [review]
LGTM

**Pull request merged**